client/src/components/DiagnosticContainer.jsx

```diff
@@ -51,10 +48,46 @@ function DiagnosticContainer() {
       </Typography>
       <hr className={classes.hr} />
       <HealthLogToggle />
+      <Dialog
```

Comment:
Could you make the background colour as the app's background colour?

Reply:
I looked a little bit farther into it. It does work but I needed to refresh. not sure yet why